### ISSUE TITLE
🐛 BUG: Heading structure

### ISSUE THREAD

**Issue by jurjendevries** [dilmaheu/dilmahtea.me#119]
**Describe the bug**
Some pages / designs don't have structured headings which isn't WCAG proof.

**Steps to Reproduce**
1. Go to https://dev.dilmahtea-me.pages.dev/en/
2. Inspect HTML
4. See the issue the headings are not structured

**Expected behavior**
For all pages the headings should be structured

**Screenshots**
N/A

**Desktop (please complete the following information):**
Desktop independent

**Smartphone (please complete the following information):**
Mobile independent

**Additional context**
Also see https://wave.webaim.org/report#/https://dilmahtea.me/en/ structure tab
Also update Figma UI designs were there is no correct structure yet.
